gems/aws-sdk-core/lib/aws-sdk-core/credential_provider_chain.rb

```diff
@@ -98,7 +98,7 @@ def shared_credentials(options)
 
     def process_credentials(options)
       profile_name = determine_profile_name(options)
-      if Aws.shared_config.config_enabled? && process_provider = Aws.shared_config.credentials_process(profile_name)
+      if Aws.shared_config.config_enabled? && process_provider = Aws.shared_config.credential_process(profile: profile_name)
```

Comment:
Rubocop didn't autocorrect this, but they recommend wrapping assignments in conditions with parenthesis

`(process_provider = Aws.shared_config.credential_process(profile: profile_name))`

Also I'd stick this on another line (too long?) in general I've been trying to push for 80 character lines.